<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Keyboard Shortcuts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>614</width>
            <height>439</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Next:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QKeySequenceEdit" name="nextKeySequenceEdit">
               <property name="keySequence">
                <string>Return</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QKeySequenceEdit" name="prevKeySequenceEdit">
               <property name="keySequence">
                <string>\</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QKeySequenceEdit" name="startKeySequenceEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Next QC Page</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QKeySequenceEdit" name="nextQBPageKeySequenceEdit">
               <property name="keySequence">
                <string>=</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Prev QC Page</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Run QC 1</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Run QC 2</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Run QC 3</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Run QC 4</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Run QC 5</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Run QC 6</string>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Run QC 7</string>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Run QC 8</string>
               </property>
              </widget>
             </item>
             <item row="14" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Run QC 9</string>
               </property>
              </widget>
             </item>
             <item row="15" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Run QC 0</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QKeySequenceEdit" name="stopKeySequenceEdit"/>
             </item>
             <item row="5" column="1">
              <widget class="QKeySequenceEdit" name="prevQBPageKeySequenceEdit">
               <property name="keySequence">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QKeySequenceEdit" name="run1KeySequenceEdit">
               <property name="keySequence">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QKeySequenceEdit" name="run2KeySequenceEdit">
               <property name="keySequence">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QKeySequenceEdit" name="run3KeySequenceEdit">
               <property name="keySequence">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QKeySequenceEdit" name="run4KeySequenceEdit">
               <property name="keySequence">
                <string>4</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QKeySequenceEdit" name="run7KeySequenceEdit">
               <property name="keySequence">
                <string>7</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QKeySequenceEdit" name="run5KeySequenceEdit">
               <property name="keySequence">
                <string>5</string>
               </property>
              </widget>
             </item>
             <item row="11" column="1">
              <widget class="QKeySequenceEdit" name="run6KeySequenceEdit">
               <property name="keySequence">
                <string>6</string>
               </property>
              </widget>
             </item>
             <item row="13" column="1">
              <widget class="QKeySequenceEdit" name="run8KeySequenceEdit">
               <property name="keySequence">
                <string>8</string>
               </property>
              </widget>
             </item>
             <item row="14" column="1">
              <widget class="QKeySequenceEdit" name="run9KeySequenceEdit">
               <property name="keySequence">
                <string>9</string>
               </property>
              </widget>
             </item>
             <item row="15" column="1">
              <widget class="QKeySequenceEdit" name="run0KeySequenceEdit">
               <property name="keySequence">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
